app/controllers/charges_controller.rb

```diff
@@ -111,9 +108,9 @@ def validate(seller_id:, project_id:, line_items:, is_distribution:)
         raise InvalidLineItem, 'line_item.quantity must be an Integer'
       end
 
-      @campaign = if is_distribution.present?
-                    # TODO(justintmckibben): Delete this case when we start using campaign_id
-                    #                        in the frontend
+      @campaign = if charge_params[:campaign_id].present? && !project_id.present?
```

Comment:
I think we can replace this whole line with:
`@campaign = Campaign.find_by(id: charge_params[:campaign_id]) if charge_params[:campaign_id].present?`